### PR TITLE
Return 404 when pod with buildId does not exist

### PR DESCRIPTION
**Pull request by d2lam**
- Use boom to wrap error 
- Return 404 when podname is not found

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -82,12 +83,13 @@ class K8sExecutor extends Executor {
...
         request.get(options, (err, resp, body) => {
             if (err) {
-                return response(new Error(`Error getting pod with sdbuild=${config.buildId}`));
+                return response(boom.wrap(err));
             }
             const podName = hoek.reach(body, 'items.0.metadata.name');
 
             if (!podName) {
-                return response(new Error(`Error getting pod name: ${JSON.stringify(body)}`));
+                return response(boom.notFound(`Pod with sdbuild=${config.buildId} does not exist:
```
Comment (stjohnjohnson) [inline index.js]: I would rather be less detailed here, something like `Unable to find build matching ${config.buildId}`

Comment (d2lam) [inline index.js]: Okay 👍

**Comment by nkatzman** [comment]
LGTM 👍

**Comment by d2lam** [comment]
Will handle this inside the model where it calls stream.

**Pull request closed without merging**